tests/test_cli.py

```diff
@@ -39,6 +42,22 @@ def test_config(tmpdir):
     init(workdir, TESTDATA_READS)
     change_config(workdir / "blr.yaml", [("read_mapper", "bwa")])
 
+# TODO add trim test for blr reads.
+
+
+def test_trim_tenx(tmpdir):
+    workdir = tmpdir / "analysis"
+    init(workdir, TESTDATA_TENX_READ1)
+    change_config(
+        workdir / DEFAULT_CONFIG,
+        [("library_type", "10x"),
+         ("barcode_whitelist", TESTDATA_TENX_BARCODES)]
+    )
+    trimmed = ["trimmed.barcoded.1.fastq.gz", "trimmed.barcoded.2.fastq.gz"]
+    run(workdir=workdir, targets=trimmed)
+    for raw, trimmed in zip((TESTDATA_TENX_READ1, TESTDATA_TENX_READ2), trimmed):
+        assert count_fastq_reads(raw) == count_fastq_reads(Path(workdir / trimmed))
```

Comment:
I'd only run this for one of the reads, since if one is created so will the other.

Reply:
Does it hurt to run them both though?

Reply:
It takes computational time, code and it's not consistent with how we run the blr test. Arguably these are more or less infinitesimal, the code is minimal but I'd wonder if it adds anything. 

Mostly I'm thinking about if we're geting any additional information for the inconsistency.

Reply:
I am not sure what inconsistency you speak of. Each test is separate from the other.

As for what is gained I think just testing that both the final files exist rather than just one is sound. Just checking read 1 is not as complete as confirming that both exist and contain the expected number of lines.